.github/workflows/build.yml

```diff
@@ -478,7 +478,7 @@ jobs:
           labels: ${{ steps.dev-rollup.outputs.labels }}
 
   test-demo:
-    if: ${{ github.event_name != 'pull_request' }}
```

Comment:
I think you can also use workflow dispatch to trigger it even though it has this if statement. Not 100% sure though.